zeebe/exporters/operate-exporter/src/main/java/io/camunda/zeebe/operate/exporter/handlers/ProcessHandler.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.zeebe.operate.exporter.handlers;
+
+import static io.camunda.zeebe.operate.exporter.util.OperateExportUtil.tenantOrDefault;
+
+import io.camunda.operate.entities.ProcessEntity;
+import io.camunda.operate.exceptions.PersistenceException;
+import io.camunda.operate.schema.indices.ProcessIndex;
+import io.camunda.operate.store.elasticsearch.NewElasticsearchBatchRequest;
+import io.camunda.zeebe.operate.exporter.util.XMLUtil;
+import io.camunda.zeebe.protocol.record.Record;
+import io.camunda.zeebe.protocol.record.ValueType;
+import io.camunda.zeebe.protocol.record.intent.Intent;
+import io.camunda.zeebe.protocol.record.intent.ProcessIntent;
+import io.camunda.zeebe.protocol.record.value.deployment.Process;
+import java.nio.charset.Charset;
+import java.nio.charset.StandardCharsets;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Optional;
+import java.util.Set;
+
+public class ProcessHandler implements ExportHandler<ProcessEntity, Process> {
+
+  private static final Set<Intent> STATES = new HashSet<>();
+
+  private static final Charset CHARSET = StandardCharsets.UTF_8;
+
+  static {
+    STATES.add(ProcessIntent.CREATED);
+  }
+
+  private final ProcessIndex processIndex;
+
+  private final XMLUtil xmlUtil = new XMLUtil();
```

Comment:
Can we possibly make this a parameter to the constructor or function that uses it? XMLUtil looks fairly complex and creating it directly in here doesn't allow us to mock in tests